06_puppy/patrickmarabeas/store_test.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type StoreSuite struct {
+	suite.Suite
+	store Storer
+}
+
+func (suite *StoreSuite) TestCreate() {
+	a := assert.New(suite.T())
+	id := suite.store.Create(Puppy{Breed: "Wolf", Color: "Grey", Value: "450"})
+	id2 := suite.store.Create(Puppy{Breed: "Boxer", Color: "Brown", Value: "300"})
+
+	a.EqualValues(id+1, id2)
+}
+
+func (suite *StoreSuite) TestRead() {
+	a := assert.New(suite.T())
+	data := suite.store.Read(0)
+
+	a.Equal(data, Puppy{ID: 0, Breed: "Wolf", Color: "Grey", Value: "450"})
+}
+
+func (suite *StoreSuite) TestReadNonExistent() {
+	a := assert.New(suite.T())
+	success := suite.store.Read(100)
```

Comment:
success seems like a strange name for this variable. Maybe `notFound` or something which indicates that it didn't exist?